tools/xcodeproj_shims/installers/_swiftmodule.sh

```diff
@@ -22,11 +26,11 @@ for index_dir in "${dirs[@]}"; do
         sourceinfo_dir="$module_bundle/Project"
         mkdir -p "$sourceinfo_dir"
 
-        cp "$module" "$module_bundle/$CURRENT_ARCH.swiftmodule" || true
-        cp "$doc" "$module_bundle/$CURRENT_ARCH.swiftdoc" || true
-        cp "$sourceinfo" "$sourceinfo_dir/$CURRENT_ARCH.swiftsourceinfo" || true
+        cp "$module" "$module_bundle/${ARCHS[0]}.swiftmodule" || true
+        cp "$doc" "$module_bundle/${ARCHS[0]}.swiftdoc" || true
+        cp "$sourceinfo" "$sourceinfo_dir/${ARCHS[0]}.swiftsourceinfo" || true
```

Comment:
FYI `CURRENT_ARCH` was always `unsupported` for me, had to use `ARCHS` and pick the first element to get the expected value (e.g. `arm64` on M1)